junifer/data/masks.py

```diff
@@ -268,20 +333,82 @@ def get_mask(
             )
         mask_img = all_masks[0]
 
+    # Warp mask if not in MNI space
+    if not target_data["space"].startswith("MNI"):
```

Comment:
This will have conflicts with other non-MNI spaces. I think we need to check for either "native" or not any standard space supported by junifer.

Reply:
I wanted to get the native check in the native support PR. About standard spaces supported by junifer, there's not really a list of them now but in that case we need to make one, would you agree to making one?

Reply:
For me it does not make much sense to have a "list" of supported at this moment. A user can create a dataset with space "XYZ" and then add a mask/parcellation for such space and it should work.

Here the condition to test is "native" and not if it's MNI. As the idea is to warp from whatever the mask is to "native" space. Now this "whatever" needs to match the available transform space.